### ISSUE TITLE
Incorrectly parse `hsl` if `hue` present in `angle`  format

### ISSUE THREAD

**Issue by Semigradsky** [omgovich/colord#34]
```ts
colord('hsl(0deg 100% 50%)')
// r { parsed: null, rgba: { r: 0, g: 0, b: 0, a: 1 } }
```

But works if `hue` in number format:
```ts
colord('hsl(0 100% 50%)')
// r {
//   parsed: { r: 255, g: 0, b: 0, a: 1 },
//   rgba: { r: 255, g: 0, b: 0, a: 1 }
// }
```

Syntax: https://www.w3.org/TR/css-color-4/#hue-syntax

**Comment by omgovich** [comment]
Good catch, thanks! Will do that this week.

**Comment by omgovich** [comment]
Fixed in v1.3.1 🚀
